compiler/luci/pass/src/ConvertNCHWToNHWCPass.cpp

```diff
@@ -100,7 +100,7 @@ luci::CircleTranspose *create_pre_transpose(luci::CircleNode *node)
 uint32_t cal_offset(const loco::TensorShape &dimension, const uint32_t *indices)
 {
   return indices[0] * dimension.dim(1).value() * dimension.dim(2).value() *
-             dimension.dim(3).value() +
+           dimension.dim(3).value() +
```

Comment:
This change is not mine. However, format-checker want to fix this format error in my PR.

Reply:
#5496 is created. I will remove this change in my PR soon.